Dockerfile

```diff
@@ -67,11 +65,16 @@ RUN \
     && ln -s $CASSANDRA_HOME /usr/local/apache-cassandra \
     && wget -q -O - https://github.com/Yelp/dumb-init/releases/download/v${DI_VERSION}/dumb-init_${DI_VERSION}_amd64 > /sbin/dumb-init \
     && echo "$DI_SHA  /sbin/dumb-init" | sha256sum -c - \
+    && adduser --disabled-password --no-create-home --gecos '' --disabled-login cassandra \
+    && mkdir -p /var/lib/cassandra/ /etc/cassandra/triggers
+
+COPY files /
```

Comment:
The problem that I was running into is that the update of of the base image and install of Java takes around 20 minutes. If it's separated into two chunks we can use the build cache and the time to update and test the image drops to below a minute. The tradeoff is additional layers vs developer ergonomics. Is there a way to reduce build times without creating new layers? In this instance the layer should be small, are the drawbacks of not having it worth it?